templates/reactioncommerce/1/rancher-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: '2'
+catalog:
+  name: "reactiondemo"
+  version: 1.4.0-rancher2
+  description: "E-commerce powered by reaction"
+  uuid: reaction-0
+  minimum_rancher_version: v0.51.0
+  questions:
+      - variable: "REACTION_HOST"
+        description: "Hostname to use for the reaction server"
+        label: "reaction Hostname"
+        required: true
+        default: "reaction"
+        type: "string"
+      - variable: "REACTION_DOMAIN"
+        description: "Domain to use for the reaction server"
+        label: "reaction Domain"
+        required: true
+        default: ""
+        type: "string"
+      - variable: host_label
+        label: "Host Label to reaction Tags"
+        description: |
+          Host label to use as reaction 'value' tag.
+          Example: 'reaction=true'
+        required: false
+        default: "private=true"
+        type: "string"
+      - variable: REACTION_EMAIL
+        label: "reaction Email"
+        required: true
+        default: ""
+        type: "string"
+      - variable: REACTION_USER
+        label: "reaction User"
+        required: true
+        default: ""
+        type: "string"
+      - variable: REACTION_AUTH
+        label: "reaction Auth"
+        required: true
+        default: ""
+        type: "password"
+      - variable: "mongo_link"
+        description: |
+          MongoDB external service link
+          cluster.
+        label: "External stack/service"
+        default: ""
+        required: true
+        type: "service"
+      - variable: "MONGO_URL"
```

Comment:
But in that case, mongo_link is required. you never get the case of don't set external link.

Anyway, if you want to use an external database, may be it would be better to use external service, and use same mongo_url...
```
  mongo:
    external_ips:
    - 1.1.1.1
    start_on_create: true
```